configure.ac

```diff
@@ -64,6 +64,8 @@ AC_SEARCH_LIBS([dlopen], [dl], [], [AC_MSG_ERROR([NCCL OFI Plugin requires dlope
 # Checks for external packages
 CHECK_PKG_LIBFABRIC([], [AC_MSG_ERROR([NCCL OFI Plugin could not find a working Libfabric install.])])
 
+CHECK_ENABLE_UBSAN()
```

Comment:
Since this is a compiler feature, it should really be around line 37.  But definitely not in between two checks for external packages.